### PR TITLE
LIVE-1533: add body story and remove interactives and richlink from r…

### PR DESCRIPTION
**Pull request by joecowton1**
## Why are you doing this?

Rich links and Interactive's aren't required in Editions.

## Changes

- remove rich links and interactives
- add article body to template stories (except media template)

### CODE REVIEW

File: src/fixtures/item.ts [hunk 1]
```diff
@@ -80,10 +81,119 @@ const mainMedia = {
...
 };
 
+const doc = new DocumentFragment();
+
+const el = document.createElement('p');
+
+el.innerText =
+	'Readers of Prospect magazine recently voted him the world’s fourth-best thinker. And right now he is thinking about 3 November, and whether the United States will reject or endorse Donald Trump. No one knows what will happen; not even West, not least because in the US he sees contradictions that even he can’t fully explain.';
+
+doc.appendChild(el);
```
Comment (faresite) [inline src/fixtures/item.ts:91]: just for Marie Kondo reasons, can we put this in a `docFixture()` function?

File: src/renderer.ts [hunk 2]
```diff
@@ -819,21 +819,9 @@ const renderEditions = (format: Format, excludeStyles = false) => (
...
 		case ElementKind.LiveEvent:
 			return h(LiveEventLink, { ...element, key });
 
-		case ElementKind.Interactive:
-			return h(Interactive, {
-				url: element.url,
-				key,
-				title: element.alt,
-			});
-
```
Comment (faresite) [inline src/renderer.ts:836]: I'm surprised TypeScript doesn't complaing that we didn't exhaust all options .. might be worth adding a default case returning `null` ?

Comment (joecowton1) [reply src/renderer.ts:836]: Good shout, updated!

File: src/fixtures/item.ts (hunk 1 above)
Comment (joecowton1) [reply src/fixtures/item.ts]: done :)

**Review by faresite — APPROVED** [review]
nice 👍

**Pull request merged**